src/datasources/system/components/SystemQueryEditor.tsx

```diff
@@ -13,13 +13,26 @@ export function SystemQueryEditor({ query, onChange, onRunQuery }: Props) {
     onRunQuery();
   }
 
+  const onIdBlur = (e: React.FocusEvent<HTMLInputElement>) => {
+    const id = e.currentTarget.value;
+    if (query.systemId !== id) {
+      onChange({ ...query, systemId: id }) //change to systemid
+      onRunQuery(); 
+    }
+  }
+
   return (
     <div>
       <InlineFieldRow >
         <InlineField label="Query type">
           <RadioButtonGroup options={enumToOptions(QueryType)} onChange={onQueryTypeChange} value={query.queryKind} />
         </InlineField>
       </InlineFieldRow>
+      <InlineFieldRow>
+        <InlineField label="Minion ID">
```

Comment:
If this field can be either the alias or the id, we should rename its label to something more generic, like "System name". We could also add the `tooltip` prop with a description of the accepted values.

Reply:
We discussed this with Mark as a "nice-to-have" in case they accidentally enter the alias instead of the id (when not using a query variable). Maybe it doesn't matter that much, but it might be confusing to users as "name" doesn't map to anything in the API. If we change the label I second adding some context help that guides the user.

Reply:
Yeah that's a fair point. We do use "name" in the Systems UI, though:

![image](https://github.com/ni/systemlink-grafana-plugins/assets/9257800/c53da82c-7420-4992-8525-f065fa979075)

I could go either way. @prestwick any input?